### PR TITLE
Tag OpenCL v0.5.0 [https://github.com/JuliaGPU/OpenCL.jl]

### PR DESCRIPTION
**Pull request by vchuravy**
Prepares OpenCL.jl for the Julia v0.5 release.

### CODE REVIEW

File: OpenCL/versions/0.5.0/requires [hunk 1]
```diff
@@ -0,0 +1 @@
+julia 0.5
```
Comment (tkelman) [inline OpenCL/versions/0.5.0/requires]: this won't be installable yet

Comment (vchuravy) [inline OpenCL/versions/0.5.0/requires]: That should be fine for now, just so that people get the right version once it is release or should I use `v0.5-`?

Comment (tkelman) [inline OpenCL/versions/0.5.0/requires]: could set one of the rc's as a minimum

Comment (vchuravy) [inline OpenCL/versions/0.5.0/requires]: That would just be `v0.5-rc0`?

**Pull request closed without merging**